feature/qos/otg_tests/egress_traffic_classification_and_rewrite_test/README.md

```diff
@@ -0,0 +1,205 @@
+
+# DP-1.17: Egress traffic DSCP rewrite
+## Summary
+
+This test validates egress traffic scheduling and packet remarking (rewrite) on a DUT. IP payload encapsulated as IPoGRE,
+IPoMPLSoGRE,IPoGUE or IPoMPLSoGUE would be decapsulated by DUT as per tunnel termination configuration and forwarded as an 
+IP payload via the egress interface. IP payload should be scheduled via egress interface according to DSCP markings prior to rewrite.
+DSCP values of IP payload encapsulated as IPoGRE, IPoMPLSoGRE, IPoGUE or IPoMPLSoGUE should have payload DSCP values re-written 
+per egress QOS re-write policy applied to the egress interface. Egress QOS policy should support match conditions on payload 
+IP header fields, specifically DSCP values. Based on the match condition, DSCP must be set to a new value. DUT configuration 
+will be evaluated against OpenConfig QOS model, and traffic flows analyzed to ensure proper scheduling, re-marking, and forwarding.
+
+## Testbed type
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+### Test environment setup
+
+* DUT has an ingress port and 1 egress port.
+
+| | [ ATE Port 1 ] ----- | DUT | ----- [ ATE Port 2 ] | |
+
+
+* Configure DUT's ingress and egress interfaces.
+
+
+#### Configuration
+
+1. DUT:Port1 is a Singleton IP interface towards ATE:Port1.
+2. DUT:Port2 is a Singleton IP interface towards ATE:Port2.
+3. DUT forms one IPv4 and one IPV6 eBGP session with ATE:Port1 using connected Singleton interface IPs.
+4. DUT forms one IPv4 and one IPV6 eBGP session with ATE:Port2 using connected Singleton interface IPs.
+5. DUT has IPv4-DST-DECAP/32 and IPv6-DST-DECAP/128 advertised to ATE:Port1 via IPv4 BGP. This IP is used for decapsulation.
+6. ATE:Port2 advertises destination networks IPv4-DST-NET/32 and IPv6-DST-NET/128 to DUT.
+7. DUT decapsulates IPoGRE, IPoMPLSoGRE,IPoGUE or IPoMPLSoGUE payload with destination of IPv4-DST-DECAP/32 and IPv6-DST-DECAP/128
+8. DUT has MPLS static forwarding rule for LabelEntry matching outer label 100 pointing to a NHG resolved via ATE:Port2.
+9. DUT matches packets on DSCP/TC values and sets new DSCP values based on remarking rules.
+
+
+* Re-marking table
+
+Forwarding Group	IPv4 TOS	IPv6 TC
```

Comment:
You can format this as a table using GFM:

```markdown
| foo | bar |
| --- | --- |
| baz | bim |
```

will render as:

| foo | bar |
| --- | --- |
| baz | bim |